torch/_refs/__init__.py

```diff
@@ -1552,6 +1579,64 @@ def reshape(a: TensorLikeType, shape: ShapeType) -> TensorLikeType:
     return _reshape_view_helper(a, shape, allow_copy=True)
 
 
+def rot90(a: TensorLikeType, k: int = 1, dims: DimsSequenceType = (0, 1)) -> TensorLikeType:
+    """Reference implementation of :func:`torch.rot90`."""
+    if len(dims) != 2:
+        raise ValueError(f"expected total rotation dims == 2, but got dims = {len(dims)}")
+    if a.ndim < 2:
+        raise ValueError(f"expected total dims >= 2, but got total dims = {a.ndim}")
+    if not (dims[0] != dims[1] and (dims[0] - dims[1]).__abs__() != a.ndim):
```

Comment:
Regular Python abs is probably more readable than the dunder here?

Reply:
I know and that's lame but we should get consistent about calling `builtins.abs` directly (and more generally calling `builtins.foo` whenever we clobber a Python name)

Reply:
Thanks! Let's use `builtins.abs`.